src/mimeparser.rs

```diff
@@ -379,6 +379,7 @@ impl MimeMessage {
         Ok(())
     }
 
+    #[allow(clippy::indexing_slicing)]
```

Comment:
Are you sure that this fn can't be written nicer? It's quite ugly currently

Reply:
Fixed this one